server/constants/server.ts

```diff
@@ -0,0 +1,5 @@
+export enum MediaServerType {
+  PLEX = 1,
+  JELLYFIN, //also works for emby (identical APIs, etc)
```

Comment:
Are the APIs really 100% identical?  Even if they are, they aren't guaranteed to remain that way.  We should have separate media server types for Jellyfin and Emby.

Reply:
I think that this PR should just add Jellyfin support, and we can look into Emby as follow-up work once we've thoroughly tested the Jellyfin stuff.

Reply:
True! I'll remove this comment, it's the only thing that mentions emby support. I'll rename this PR as well.

Reply:
You also need to update the PR description.  Your PR fixes #288, not #295.